go.mod

```diff
@@ -1,38 +1,42 @@
 module google.golang.org/grpc
 
-go 1.22
+go 1.22.0
+
+toolchain go1.23.4
```

Comment:
How did this get added? Is is due to a dependency specifying this or does it come from your locally installed toolchain?

Reply:
We need to support go.1.22 until go.24 is released. I think we should restrict the `go` and `toolchain` directives to use strictly less than 1.23.